### PR TITLE
feat(settings): team index is the member list

### PR DESCRIPTION
**Pull request by Chrissy**
This doesn't address everything (I need to de-table the members list) but i wanna get this out while it's still smallish:

image

### CODE REVIEW

**Review by macqueen — APPROVED** [review]

File: src/sentry/static/sentry/app/components/userBadge.jsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import Avatar from './avatar';
+import Link from './link';
+
+const UserBadge = ({user, orgId}) => {
```
Comment (billyvg) [inline src/sentry/static/sentry/app/components/userBadge.jsx]: Is this something we can use for Org members list as well?

File: src/sentry/static/sentry/app/views/settings/team/teamMembers.jsx [hunk 2]
```diff
@@ -96,6 +104,26 @@ const TeamMembers = createReactClass({
...
+      <Button
+        priority="primary"
+        size="small"
+        className="pull-right"
+        to={`/settings/organization/${orgId}/members/new/`}
+      >
+        <span className="icon-plus" /> {t('Invite Member')}
+      </Button>
+    ) : (
+      <a
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/settings/team/teamMembers.jsx]: Why no `<Button>` here?

File: src/sentry/static/sentry/app/views/settings/team/teamMembers.jsx [hunk 3]
```diff
@@ -96,6 +104,26 @@ const TeamMembers = createReactClass({
...
+        className="btn btn-primary btn-sm btn-disabled tip pull-right"
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/settings/team/teamMembers.jsx]: Use `<Tooltip>`

File: src/sentry/static/sentry/app/views/teamDetails.jsx [hunk 4]
```diff
@@ -89,33 +87,20 @@ const TeamDetails = createReactClass({
...
 
     //TODO(maxbittker) remove hack to not show this page on old settings
     let onNewSettings = routePrefix.startsWith('/settings/');
 
-    const features = new Set(this.context.organization.features);
     return (
       <div>
         <h3>{team.name}</h3>
 
-        {!features.has('new-settings') &&
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/teamDetails.jsx]: Doesn't this mean for all users, they won't be able to remove a team?

iirc Remove Team in Team Settings is only for users with `new-settings` feature tag.

File: src/sentry/static/sentry/app/components/userBadge.jsx (hunk 1 above)
Comment (Chrissy) [reply src/sentry/static/sentry/app/components/userBadge.jsx]: yeah that's the idea

File: src/sentry/static/sentry/app/views/settings/team/teamMembers.jsx (hunk 2 above)
Comment (Chrissy) [reply src/sentry/static/sentry/app/views/settings/team/teamMembers.jsx]: great question. this code is just copied from the old stuff, but i'll investigate as I continue working on it.

File: src/sentry/static/sentry/app/views/teamDetails.jsx (hunk 4 above)
Comment (Chrissy) [reply src/sentry/static/sentry/app/views/teamDetails.jsx]: yeah this is weird. i'm not sure why this dropdown was even appearing for me, but I will figure out what is going on and resolve it immediately

Comment (Chrissy) [reply src/sentry/static/sentry/app/views/teamDetails.jsx]: (my assumption was that is was a leftover feature duplication from before we added the delete button to team settings)

**Comment by Chrissy** [comment]
@billy this already went out (might have had a bit of an itchy trigger finger on the big green button) but I think we should be ok without reverting so long as I get on this now. I'll work on all of these suggestions asap, especially the one about the delete button.

**Pull request merged**